Resources/public/js/views/ez-universaldiscoveryview.js

```diff
@@ -518,6 +520,16 @@ YUI.add('ez-universaldiscoveryview', function (Y) {
             },
 
             /**
+             * The depth of the root where we start discovering content
```

Comment:
this should explain that this parameter is related to the starting Location. Without a starting Location or with a value greater than the depth of the starting Location, this is useless.